README.md

```diff
@@ -22,24 +22,24 @@ This project was generated using [Nx](https://nx.dev).
 
 ### Alpha Status Implications
 
--   Alpha testers are needed to guide the implementation requirements needed for production deployments
--   As a pre-release, the API will likely change -- be prepared for breaking changes as we develop the optimal solution
+- Alpha testers are needed to guide the implementation requirements needed for production deployments
+- As a pre-release, the API will likely change -- be prepared for breaking changes as we develop the optimal solution
```

Comment:
Is this still accurate after this is merged?

Reply:
I think no, but I will recheck